SimPEG/Directives.py

```diff
@@ -343,14 +356,14 @@ def endIter(self):
             else:
                 self.f_old = phim_new
 
-            # Cool the threshold parameter if required
-            if getattr(self, 'factor', None) is not None:
-                eps = self.reg.eps / self.factor
-
-                if getattr(self, 'eps_min', None) is not None:
-                    self.reg.eps = np.max([self.eps_min,eps])
-                else:
-                    self.reg.eps = eps
+#            # Cool the threshold parameter if required
```

Comment:
We might want to explore the cooling schedule later when doing the non-linear stuff. So lets keep it for now.